Source/Core/DolphinQt2/Config/Graphics/GraphicsWindow.cpp

```diff
@@ -0,0 +1,52 @@
+// Copyright 2017 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include "DolphinQt2/Config/Graphics/GraphicsWindow.h"
+#include "DolphinQt2/Config/Graphics/AdvancedWidget.h"
+#include "DolphinQt2/Config/Graphics/EnhancementsWidget.h"
+#include "DolphinQt2/Config/Graphics/GeneralWidget.h"
+#include "DolphinQt2/Config/Graphics/HacksWidget.h"
+
+#include <QDialogButtonBox>
+#include <QTabWidget>
+#include <QVBoxLayout>
+
+GraphicsWindow::GraphicsWindow(QWidget* parent, X11Utils::XRRConfiguration* xrr_config)
+    : QDialog(parent), m_xrr_config(xrr_config)
+{
+  CreateMainLayout();
+  ConnectWidgets();
+
+  setWindowTitle(tr("Graphics"));
+  setWindowFlags(windowFlags() & ~Qt::WindowContextHelpButtonHint);
```

Comment:
Mainly I was thinking that this isn't actually a dialog (none of these settings windows are). A dialog is intended for brief communication (questions, information, progress), not a full configuration window.